release/e2e.py

```diff
@@ -1338,6 +1338,12 @@ def run_test_config(
 
     app_config_rel_path = test_config["cluster"].get("app_config", None)
     app_config = _load_config(local_dir, app_config_rel_path)
+    if test_config.get("app_env_vars") is not None:
```

Comment:
Little confused what this is for?

Reply:
Yep, let me add a comment for this.
Basically, for a lot of staging test, they are the same as previous ones, except the flags. This is to include the flags in test suites when we define it so that I don't need to create app config for each one of them.